command/base_flags.go

```diff
@@ -374,9 +382,12 @@ func (i *uintValue) Set(s string) error {
 	if err != nil {
 		return err
 	}
+	if v > 0 && v <= uint64(MaxUint) {
```

Comment:
Doesn't this have to be `v >= 0`?  Otherwise, if someone provides a value of 0, we'll return an error, right?

Sorry I missed this in your original PR, I guess if I'm right we'll need to fix on main too.